### ISSUE TITLE
Ticking entity crash from modded mobs

### ISSUE THREAD

**Issue by GwydionPwyll** [Shadows-of-Fire/Apotheosis#623]
Receiving a ticking entity crash when [edit: simply being near] the EvilCraft Nether Silverfish. Error indicates it can't find the apotheosis:current_hp_damage attribute.

```
[12:26:29] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.ReportedException: Ticking entity
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:906) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:842) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.client.server.IntegratedServer.m_5705_(IntegratedServer.java:84) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:685) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.server.MinecraftServer.m_177918_(MinecraftServer.java:261) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at java.lang.Thread.run(Unknown Source) [?:?]
Caused by: java.lang.IllegalArgumentException: Can't find attribute apotheosis:current_hp_damage
	at net.minecraft.world.entity.ai.attributes.AttributeSupplier.m_22260_(AttributeSupplier.java:21) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.ai.attributes.AttributeSupplier.m_22245_(AttributeSupplier.java:28) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.ai.attributes.AttributeMap.m_22181_(AttributeMap.java:62) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.LivingEntity.m_21133_(LivingEntity.java:1818) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at shadows.apotheosis.adventure.AdventureModuleEvents.attack(AdventureModuleEvents.java:199) ~[Apotheosis-1.18.2-5.4.1.jar%2371!/:5.4.1]
	at net.minecraftforge.eventbus.ASMEventHandler_2701_AdventureModuleEvents_attack_LivingAttackEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-5.0.3.jar%232!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-5.0.3.jar%232!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-5.0.3.jar%232!/:?]
	at net.minecraftforge.common.ForgeHooks.onPlayerAttack(ForgeHooks.java:307) ~[forge-1.18.2-40.1.68-universal.jar%23388!/:?]
	at net.minecraft.world.entity.player.Player.m_6469_(Player.java:806) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.server.level.ServerPlayer.m_6469_(ServerPlayer.java:664) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.Mob.m_7327_(Mob.java:1326) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at org.cyclops.evilcraft.entity.monster.EntityNetherfish.m_7327_(EntityNetherfish.java:48) ~[EvilCraft-1.18.2-1.2.6.jar%23166!/:1.2.6]
	at net.minecraft.world.entity.ai.goal.MeleeAttackGoal.m_6739_(MeleeAttackGoal.java:147) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.ai.goal.MeleeAttackGoal.m_8037_(MeleeAttackGoal.java:138) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.ai.goal.WrappedGoal.m_8037_(WrappedGoal.java:65) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.ai.goal.GoalSelector.m_186081_(GoalSelector.java:119) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.Mob.m_6140_(Mob.java:691) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.LivingEntity.m_8107_(LivingEntity.java:2506) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.Mob.m_8107_(Mob.java:495) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.monster.Monster.m_8107_(Monster.java:41) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at org.cyclops.evilcraft.entity.monster.EntityNetherfish.m_8107_(EntityNetherfish.java:58) ~[EvilCraft-1.18.2-1.2.6.jar%23166!/:1.2.6]
	at net.minecraft.world.entity.LivingEntity.m_8119_(LivingEntity.java:2253) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.Mob.m_8119_(Mob.java:307) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.entity.monster.Silverfish.m_8119_(Silverfish.java:101) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.server.level.ServerLevel.m_8647_(ServerLevel.java:652) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.level.Level.m_46653_(Level.java:2574) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.server.level.ServerLevel.m_184063_(ServerLevel.java:319) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.world.level.entity.EntityTickList.m_156910_(EntityTickList.java:54) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:299) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:902) ~[client-1.18.2-20220404.173914-srg.jar%23383!/:?]
	... 5 more
```

**Comment by tharrmeehan** [comment]
I'm having a similiar issue to a ticking entity, but it seems to be with a shulker from a datapack.

Crash log:
https://gist.github.com/tharrmeehan/4b123bf0d211e0a01d01ca0a74e3c5f9

**Comment by Kangy1103** [comment]
I am also having this issue except I was attacking 2 vanilla zombies

[crash-2022-08-20_21.29.52-server.txt](https://github.com/Shadows-of-Fire/Apotheosis/files/9388041/crash-2022-08-20_21.29.52-server.txt)

**Comment by Shadows-of-Fire** [comment]
So all three of these logs are different issues, though they should be fixed for 5.5.0
(Except @Kangy1103 's issue, which is just a server timeout, and not a mod problem)

**Comment by Kangy1103** [comment]
> So all three of these logs are different issues, though they should be fixed for 5.5.0
> (Except @Kangy1103 's issue, which is just a server timeout, and not a mod problem)

My mistake, seems I posted the wrong crash report 🤦🏼
